cranelift/filetests/filetests/interpreter/extractlane.clif

```diff
@@ -0,0 +1,23 @@
+test interpret
```

Comment:
Trying this I'm getting errors that `i32x4` is an unsupported type for `uadd_sat` and `usub_sat` on the `test run` side -- should that be the case?

Reply:
Hmm, according to the [docs](https://docs.rs/cranelift-codegen/0.76.0/cranelift_codegen/ir/trait.InstBuilder.html#method.uadd_sat), it should work.

However, the reason that it probably doesn't work is that its not included in the [WASM SIMD Spec](https://github.com/WebAssembly/simd/blob/main/proposals/simd/SIMD.md#saturating-integer-addition).

What we usually do in these cases is separate the testcases in two files. `8x16` / `16x8` in one file with the run targets, and `i32x4` / `i64x2` (could you add testcases for `i64x2` as well?) into another with just the interpreter.

I also usually file an issue stating that these are not implemented in the x86_64 and aarch64 backends. We've been finding a lot of code that should compile but doesn't since we started adding these tests instead of relying on the wasm test suite.